src/app/issues-viewer/card-view/card-view.component.html

```diff
@@ -28,5 +28,10 @@
       </mat-card>
     </div>
   </div>
-  <mat-paginator [pageSize]="20" [pageSizeOptions]="[10, 20, 50]"></mat-paginator>
+  <mat-paginator
+    [pageSize]="pageSize"
+    [pageSizeOptions]="[10, 20, 50]"
+    [class]="pageSize > issueLength ? 'pagination-hide-arrow' : ''"
```

Comment:
Should the comparator be ``>`` or ``>=``?
![image](https://github.com/CATcher-org/WATcher/assets/88131400/64d13cf1-a35e-4973-a51d-bf94174a9123)
I believe a page size of the same number as the number of issues should hide the pagination arrows.

Reply:
Fixed!